test/properties_test.cpp

```diff
@@ -44,23 +44,42 @@ TEST(Properties, Precision) {
   EXPECT_FLOAT_EQ(cube.GetEpsilon(), 100 * kPrecision);
 }
 
-TEST(Properties, Precision2) {
+TEST(Properties, Epsilon2) {
   Manifold cube = Manifold::Cube();
   cube = cube.Translate({-0.5, 0, 0}).Scale({2, 1, 1});
   EXPECT_FLOAT_EQ(cube.GetEpsilon(), 2 * kPrecision);
 }
 
-TEST(Properties, Precision3) {
-  Manifold cylinder = Manifold::Cylinder(1, 1, 1, 1000);
-  const auto prop = cylinder.GetProperties();
-
-  MeshGL mesh = cylinder.GetMeshGL();
-  mesh.faceID.clear();
-  Manifold cylinder2 = Manifold(mesh).SetEpsilon(0.001);
-
-  const auto prop2 = cylinder2.GetProperties();
-  EXPECT_NEAR(prop.volume, prop2.volume, 0.001);
-  EXPECT_NEAR(prop.surfaceArea, prop2.surfaceArea, 0.001);
+TEST(Properties, Tolerance) {
+  double degrees = 1;
+  double tol = sind(degrees);
+  Manifold cube = Manifold::Cube({1, 1, 1}, true);
+  Manifold imperfect = (cube ^ cube.Rotate(degrees)).AsOriginal();
+  const auto prop = imperfect.GetProperties();
+
+  Manifold imperfect2 = imperfect.SetTolerance(tol);
+  MeshGL mesh = imperfect.GetMeshGL();
+  mesh.tolerance = tol;
+  Manifold imperfect3(mesh);
+
+  EXPECT_EQ(imperfect.NumTri(), 28);
+  EXPECT_EQ(imperfect2.NumTri(), 16);  // TODO: should be 12
+  EXPECT_EQ(imperfect3.NumTri(), 22);  // TODO: should be 12
```

Comment:
Our simplification is obviously failing here, but I don't think it's due to the tolerance work in this PR (e.g. no change with a larger value). I think I'll merge this for now and come back and work on our simplification later.